downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="rhdh-backup-operator-configmap_{context}"]
+= Backing up your {RHDHShort} Operator ConfigMap
+
+Before you install {AAPRHDH}, create a local copy of the ConfigMap for the {RHDHShort} Operator.
+You can use a section of the ConfigMap when you are populating a custom ConfigMap.
+
+.Procedure
+
+// Is export KUBECONFIG=/home/secrets/rosa/kubeconfig needed?
+
+. Find the namespace for your {RHDHShort} Operator.
++
+When you installed the {RHDHShort} Operator, a namespace was created for it.
+Select *Topology* and look for the {RHDHShort} Operator in the *Project* dropdown list.
+The default namespace is `rhdh-operator`.
+. Run the following command to make a copy of the ConfigMap for your {RHDHShort} Operator, `backstage-default-config`.
++
+----
+$ oc get configmap backstage-default-config -n <rhdh-operator-namespace> -o yaml > <CopyOfRhdhOperatorConfig>
+----
++
+Replace `<rhdh-operator-namespace>` with your {RHDHShort} Operator namespace, and `<CopyOfRhdhOperatorConfig>` with
+the filename you want to use for your copy of the {RHDHShort} Operator.
+
+
+.Verification
```

Comment:
Should this have "//" too? Oddly enough it doesn't show in the preview, but according to GH its not commented out?